tests/functional/models/custom-lambda/2015-03-31/paginators-1.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
What was the reasoning for pulling in the `paginators-1.json` and `examples-1.json` as well? It looks like we are not doing any assertions around there functionality.

Reply:
Primarily just to ensure we're using a consistent snapshot of the all the models.  I'd not want us to load the service model from this directory and the paginators from the built in directory.

If you want, I an add some sort of assertion on getting paginators from the clients to ensure we exercise that code.

The examples-1.json however, I don't feel that strongly about.

Reply:
I think it would make sense to add a test for paginators. I do not care to strongly about the `examples-1.json`. We can keep it there and not put any assertions on it. Do we want to add an empty `waiters-2.json` file? We may run into the same problem of loading from the builtin directory if waiters are added in the future for `Lambda`.